lib/csv_row_model/model/columns.rb

```diff
@@ -20,6 +20,17 @@ def column_names
           columns.keys
         end
 
+        # @return [Array] column friendly names for the row model
+        def friendly_names
+          @friendly_names ||= begin
+            names = []
+            columns.each do |name, options|
+              names << (options[:friendly_name] || name.to_s.titleize)
```

Comment:
i thought the same about the default too, but ultimately thought an overridable function is best (like `format_cell`), so we don't need to dig into the gem for do a mass change.